tests/e2e/e2e_test.go

```diff
@@ -72,7 +72,7 @@ var _ = Describe("odoe2e", func() {
 			})
 
 			It("Should fail if user tries to create anything other than project", func() {
-				runCmdShouldPass(fmt.Sprintf("odo login -u %s -p %s", "odoNoProjectAttemptsCreate", loginTestUserPassword))
```

Comment:
What is wrong with ```odoNoProjectAttemptsCreate```. You can use the same naming in ```configure-installer-tests-cluster.sh```.

Reply:
It is highly not recommended to use caps in in OpenShift names. Display name is where you can do that :)

Reply:
Right